agent/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghcr.io/openwallet-foundation/acapy-agent:py3.12-1.2.0
```

Comment:
Might be worth bumping the image to the latest `1.2.x` release

Reply:
Will do, thanks